src/extras/geometries/EdgesGeometry.js

```diff
@@ -86,6 +86,8 @@ THREE.EdgesGeometry = function ( geometry, thresholdAngle ) {
 
 	}
 
+	coords = coords.subarray( 0, index );
```

Comment:
I have not tested it, but I was refering to @benaadams's suggestion to "do something simpler". What do you think?